res/values/arrays.xml

```diff
@@ -2,74 +2,74 @@
 <resources>
   <string-array name="pref_layout_values">
     <item>none</item>
-    <item>azerty</item>
-    <item>qwerty</item>
-    <item>qwerty_pt</item>
-    <item>qwerty_es</item>
-    <item>qwerty_el</item>
-    <item>qwerty_ko</item>
-    <item>qwerty_lv</item>
-    <item>qwerty_no</item>
-    <item>qwerty_sv_se</item>
-    <item>qwerty_hu</item>
-    <item>qwerty_tr</item>
-    <item>qwerty_pl</item>
-    <item>qwerty_vi</item>
-    <item>ru_jcuken</item>
-    <item>qwertz</item>
-    <item>qwertz_cs</item>
-    <item>qwertz_de</item>
-    <item>qwertz_hu</item>
-    <item>qwertz_sk</item>
-    <item>bgph1</item>
-    <item>dvorak</item>
-    <item>colemak</item>
-    <item>neo2</item>
-    <item>bone</item>
-    <item>jcuken_ua</item>
-    <item>bangla</item>
-    <item>devanagari_1</item>
-    <item>devanagari_2</item>
-    <item>he_il_1452_1</item>
-    <item>he_il_1452_2</item>
-    <item>ar_pc</item>
-    <item>ar_alt</item>
+    <item>arab_ara_ibm</item>
+    <item>arab_ara_ibm_ma</item>
+    <item>beng_ben_rh</item>
+    <item>cyrl_bul_yaverti</item>
+    <item>cyrl_ukr_jcuken</item>
+    <item>cyrl_rus_jcuken</item>
+    <item>deva_hin_inscript</item>
+    <item>deva_hin_lv</item>
+    <item>grek_ell_qwerty</item>
+    <item>hebr_heb_si_1452_1</item>
+    <item>hebr_heb_si_1452_2</item>
+    <item>kore_kor_ks_x_5002</item>
+    <item>latn_ces_qwertz</item>
+    <item>latn_deu_bone</item>
+    <item>latn_deu_neo2</item>
+    <item>latn_deu_qwertz</item>
+    <item>latn_eng_colemak</item>
+    <item>latn_eng_dvorak</item>
+    <item>latn_eng_qwerty</item>
+    <item>latn_eng_qwertz</item>
```

Comment:
This layout is not English. It's German and is a variation of `latn_deu_qwertz`. `qwertz` should be renamed to `latn_deu_qwertz` and `qwertz_de` to `latn_deu_qwertz_2`.

Reply:
Hmm, my original idea was, äöüß are supposed to be on the key0s according to DIN 2137-1 for `qwertz`. It can be perceived as a generic `qwertz` with German support.

Now I think about it, DIN 2137-1 is not the factor that determines whether the keyboard is German or not.

I am cautious about simply naming it `2` because it is not intuitive which one is the number. Maybe adding more letters and diacritics to make it a real generic `qwertz`?

Reply:
That's a good point, thanks! Knowing that, `latn_eng_qwertz` works for me.

Reply:
Can you write the name of the standards you researched into the `.xml` file ? As comments like this:
```
<!-- Defined in standard DIN 2137-1 -->
```
(some layouts are not defined in standards)

Reply:
Yes, I can do it.

Reply:
That's useful, thanks! Don't put too much time on this, I just want the information you already have researched for future reference.

Reply:
Alright. I will focus on other aspects.